client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -53,13 +56,32 @@ const DisputeAwaitingResponseDetails: React.FC< Props > = ( {
 	const dueBy = moment.unix( dispute.evidence_details?.due_by ?? 0 );
 	const countdownDays = Math.floor( dueBy.diff( now, 'days', true ) );
 	const hasStagedEvidence = dispute.evidence_details?.has_evidence;
+	const { createErrorNotice } = useDispatch( 'core/notices' );
 	// This is a temporary restriction and can be removed once steps and actions for inquiries are implemented.
-	const showDisputeStepsAndActions = ! isInquiry( dispute );
+	const showDisputeSteps = ! isInquiry( dispute );
 
 	const onModalClose = () => {
 		setModalOpen( false );
 	};
 
+	const viewOrder = () => {
+		if ( orderDetails?.url ) {
+			window.location.href = orderDetails?.url;
+			return;
+		}
+
+		createErrorNotice(
+			__(
+				'Unable to view order. Order not found.',
+				'woocommerce-payments'
+			)
+		);
+	};
+
+	const challengeButtonDefaultText = ! isInquiry( dispute )
+		? __( 'Challenge dispute', 'woocommerce-payments' )
+		: __( 'Submit Evidence', 'woocommerce-payments' );
+
```

Comment:
Minor suggestion: use `isInquiry` rather than `! isInquiry` to improve readability and remain consistent with `isInquiry` checks below.

Also I suggest only capitalising the first word (`Submit evidence`) to remain consistent with `Challenge dispute`.
```suggestion
	const challengeButtonDefaultText = isInquiry( dispute )
		? __( 'Submit evidence', 'woocommerce-payments' )
		: __( 'Challenge dispute', 'woocommerce-payments' );

```